src/components/BadgeCircle.js

```diff
@@ -0,0 +1,62 @@
+import { Badge } from '@material-ui/core';
```

Comment:
We should check behaviour of badge with 2-digit numbers

Reply:
You can edit html and have a look how it looks like. We can apply different styles for two digits or we can enlarge circle as well so it fit better once there are two digits..